README.md

```diff
@@ -522,6 +522,18 @@ dsn: snowflake://user:password@myaccount/mydb/myschema
 
 See also: https://pkg.go.dev/github.com/snowflakedb/gosnowflake
 
+**MongoDB:**
+
+``` yaml
+# .tbls.yml
+dsn: mongodb://localhost:27017/
```

Comment:
How about making dbName required and using `mongodb://localhost:27017/dbName` ?

If MongoDB always has separate db's, I think it would be better for tbls to be able to document per db.

If there are any use cases where the db specification should not be required, please let me know.

Reply:
fixed in [ef2500e](https://github.com/k1LoW/tbls/pull/339/commits/ef2500ee5e9ebfd5c2155e9ba5642f66118109a4)
there is a code duplication between [test](https://github.com/k1LoW/tbls/pull/339/commits/ef2500ee5e9ebfd5c2155e9ba5642f66118109a4#diff-fe34c47009be115c37a91350041127c327d6269241a376ab7dfef867e2ca4584R17) and main logic implementation. I am not sure how to avoid such thing. Any ideas?